src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/ProgressBar/ProgressBarHandler.Android.cs

```diff
@@ -0,0 +1,13 @@
+﻿using static Android.Resource;
+using AProgressBar = Android.Widget.ProgressBar;
```

Comment:
A bit nitpicky but I think we should try and use more detailed aliases here as it took me a minute to figure out what this was.  Perhaps we can think of a convention around this `PlatformX`, or just even use the platform name `AndroidProgressBar`.

Reply:
Renamed to `AndroidProgressBar`